### ISSUE TITLE
Support for serie novels

### ISSUE THREAD

**Issue by Ellipsine** [vkbo/novelWriter#967]
It would be interesting to add support for serie of novels in Novel Writer. 

For example, I make a serie of five books inside the same project. As documentation, characters, point of view and all are the same for them, I write them in the same project. However, right now, NovelWriter considerates them as one book, not five. 

In a preview thread, you suggested to add the possibility to create multiple novel items inside a project, and I think it would be perfect.

**Comment by vkbo** [comment]
Adding this would primarily affect the build tool, so I need to implement this when I do all the other build tool changes. Since I have a bunch of requests related to the build tool, I think I will make that the primary focus of the next release, 1.7.

**Comment by jyhelle** [comment]
I second this one ! 

So far I made a duplicate of the first book when beginning the second one and deleted all the text chapters. When it is completed I'll again make a duplicate of the second project to create the third (so additional characters, locations, etc, are carried along)... Obviously that's not good for someone writing all titles in parallel !

**Comment by Ellipsine** [comment]
> Adding this would primarily affect the build tool, so I need to implement this when I do all the other build tool changes. Since I have a bunch of requests related to the build tool, I think I will make that the primary focus of the next release, 1.7.

It will be perfect, thank you :) 

> I second this one !

So far I made a duplicate of the first book when beginning the second one and deleted all the text chapters. When it is completed I'll again make a duplicate of the second project to create the third (so additional characters, locations, etc, are carried along)... Obviously that's not good for someone writing all titles in parallel !

Yes, indeed ! I would add that changing project everytime for a new novel could be a pain if we need to use a global search to check a point of detail or an information while writing.

**Comment by vkbo** [comment]
I've made significant changes to how project items are handled, which now includes making multiple root folders of the same kind. That is, you can add more than one Novel folder. However, in order for them to show up as separate novels in the Outline tab, you must add a title page to each with a level one title.

I will make a second set of changes to make them more visually distinct. But since the change added now was so large, I wanted to do it in two steps. The changes I made now are described in #1031, although they are quite technical. They are mostly my notes so I know what I did. :)

**Comment by jyhelle** [comment]
Thanks Veronica, a nice Easter gift ! I'll try that shortly since I have a real need for it

**Comment by vkbo** [comment]
I would advice against using it for an active project until I release at least 1.7 Beta 1. Mainly because some of the changes under the hood are irreversible. The project file format is changed a great deal for instance. If you open a project with this version, you cannot open it in the earlier, stable versions any more.

Testing it on a sample or test project is of course fine.

**Comment by jyhelle** [comment]
Oh yes, for sure I won't use it for production, but as I happen to have at hand an actual project I don't need to create test data for some hands on tests
--
Envoyé de l'application Mail.ru pour Android dimanche, 17 avril 2022, 09:07PM +02:00 de Veronica Berglyd Olsen  ***@***.*** :

**Comment by vkbo** [comment]
Great! Let me know if you have any issues :)
